docs/alternatives/navigator_extension_alternative.bs

```diff
@@ -45,6 +45,41 @@ Markup Shorthands: markdown yes, css no
 
 </div>
 
+# Key terms # {#key-terms}
+
+The <dfn for="web environment">web environment</dfn> is defined as <i>TODO</i>
+
+# Attesters # {#attester}
+
+The term <dfn for="attester">attester</dfn> refers to a third party capable of returning an Web
+Environment Integrity verdict. A <dfn for="verdict">Web Environment Integrity verdict</dfn> refers
```

Comment:
nit: Make `Environment Integrity verdict` linkable

Reply:
Interesting, I had trouble linking this when the dfn was just `verdict`, but it works fine when it is `Integrity verdict`.